sdk/core/src/client_request_id.rs

```diff
@@ -0,0 +1,28 @@
+use crate::headers::*;
+use crate::AddAsHeader;
+use http::request::Builder;
+use std::borrow::Cow;
+
+#[derive(Debug, Clone)]
+pub struct ClientRequestId<'a>(Cow<'a, str>);
```

Comment:
I personally don't see a point of using `Cow` here. If this is simply to save users the convenience of not writting `&` before an owned string that they want to pass to `ClientRequestId::new`, that is very much not worth the complexity of the type signature that they will see when reading the docs.

Reply:
Good point! Changed!